docs/source/notebooks/power_analysis.ipynb

```diff
@@ -0,0 +1,1412 @@
+{
```

Comment:
<p>We can change "R2" to "$R^2$ for nice math rendering.</p><p> </p><p>Consider adding links to wikipedia definitions of both metrics.</p><p> </p><p>Also consider adding a reference to a paper for Bayesian Tail Probability. Should be able to do that in references.bib.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-labs/CausalPy/pull/277/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='36'/>

Reply:
Done!